src/helpers/methods.js

```diff
@@ -11,4 +13,17 @@ const sliderChoice = event => {
   }
 };
 
-export { sliderChoice }
\ No newline at end of file
+const objectEraser = async (component, trip, restaurant) => {
+  try {
+    const response = await axios({
+      method: "DELETE",
+      url: `api/v1/${component}`,
+      params: { trip: trip, activity_type: restaurant }
+    });
+    return response;
+  } catch (error) {
+    return error;
+  }
+};
```

Comment:
Is there a reason this is here and not with the other request-methods in the  now-not-so-greatly-named module/destination.js?

Reply:
look above